src/main/groovy/com/blackducksoftware/integration/hub/detect/DetectProperties.groovy

```diff
@@ -102,9 +106,9 @@ class DetectProperties {
     @Value('${blackduck.hub.proxy.password}')
     String hubProxyPassword
 
-    @ValueDescription(description="If true the Hub https certificate will be automatically imported", defaultValue="false", group=DetectProperties.GROUP_HUB_CONFIGURATION)
-    @Value('${blackduck.hub.auto.import.cert}')
-    Boolean hubAutoImportCertificate
+    @ValueDescription(description="If true, automatically trust the certificate for the current run of Detect only", defaultValue="false", group=DetectProperties.GROUP_HUB_CONFIGURATION)
+    @Value('${blackduck.hub.always.trust.server.cert}')
```

Comment:
Can you change the property name to `blackduck.hub.trust.cert` please?

Reply:
Done and done